charts/enterprise-metrics/capped-large.yaml

```diff
@@ -0,0 +1,144 @@
+# These values configure the Grafana Enterprise Metrics cluster to
+# handle production ingestion of ~10M active series using the blocks
+# storage engine.
+# Query requirements can vary dramatically depending on query rate and query
+# ranges. The values here satisfy a "usual" query load as seen from our
+# production clusters at this scale.
+# It is important to ensure that you run no more than one ingester replica
+# per node so that a single node failure does not cause data loss. Zone aware
+# replication can be configured to ensure data replication spans availability
+# zones. Refer to [Zone Aware Replication](https://cortexmetrics.io/docs/guides/zone-aware-replication/)
+# for more information.
+# Minio is no longer enabled and you are encouraged to use your cloud providers
+# object storage service for production deployments.
+admin_api:
+  replicas: 3
+  resources:
+    requests:
+      cpu: 1
+      memory: 256Mi
+    limits:
+      cpu: 1
+      memory: 256Mi
+
+alertmanager:
+  persistentVolume:
+    enabled: true
+  replicas: 3
+  resources:
+    requests:
+      cpu: 1
+      memory: 8Gi
+    limits:
+      cpu: 1
+      memory: 8Gi
+  statefulset:
+    enabled: true
+
+compactor:
+  persistentVolume:
+    size: 50Gi
+  resources:
+    requests:
+      cpu: 2
+      memory: 2Gi
+    limits:
+      cpu: 2
+      memory: 2Gi
+
+consul:
+  resources:
+    requests:
+      cpu: 4
+      memory: 4Gi
+  server:
+    replicas: 3
+    bootstrapExpect: 3
+
+distributor:
+  replicas: 15
+  resources:
+    requests:
+      cpu: 2
+      memory: 4Gi
+    limits:
+      cpu: 2
+      memory: 4Gi
+
+gateway:
+  replicas: 8
+  resources:
+    requests:
+      cpu: 1
+      memory: 384Mi
+    limits:
+      cpu: 1
+      memory: 384Mi
+
+ingester:
+  persistentVolume:
+    size: 50Gi
+  replicas: 25
+  resources:
+    requests:
+      cpu: 4
+      memory: 25Gi
+    limits:
+      cpu: 4
+      memory: 25Gi
+
+memcached:
+  enabled: true
+  replicaCount: 2
+
+memcached-queries:
+  enabled: true
+  replicaCount: 3
+
+memcached-metadata:
+  enabled: true
+
+minio:
+  enabled: false
+
+querier:
+  replicas: 6
+  resources:
+    requests:
+      cpu: 2
+      memory: 24Gi
+    limits:
+      cpu: 2
+      memory: 24Gi
+
+query_frontend:
+  replicas: 2
+  resources:
+    requests:
+      cpu: 2
+      memory: 6Gi
+    limits:
+      cpu: 2
+      memory: 6Gi
+
+ruler:
+  replicas: 2
+  resources:
+    requests:
+      cpu: 4
+      memory: 16Gi
+    limits:
+      cpu: 4
+      memory: 16Gi
+
+store_gateway:
+  persistentVolume:
+    size: 50Gi
+  replicas: 1
+  resources:
+    requests:
+      cpu: 2
```

Comment:
Similar question regarding changes from `large.yaml`. `1` ->  `2`.

Reply:
I'll go through the replicas values again!